### PR TITLE
added files from local

### PR DESCRIPTION
**Pull request by ghangas-ds**
#### What's this PR do?
Adds files from Local to the Data team repo

#### Where should the reviewer start?
Review what was added and let me know if any more context on these files would be helpful. We can also set some internal standards around comments to make sure these can be used.

### CODE REVIEW

**Comment by ghangas-ds** [comment]
@kkennovin i updated all sql files to include a link to trello or info on where to find context. Let me know if there are other things you think are essential before this can be approved

File: Code - AdHoc Requests/2020/trello-gotv-survey-sql [hunk 1]
```diff
@@ -0,0 +1,44 @@
+# Trello request https://trello.com/c/92a9bU5j/1676-gotv-survey-match-northstars-to-campaigns
+
+
+### SQL 1
```
Comment (kkennovin) [inline Code - AdHoc Requests/2020/trello-gotv-survey-sql:4]: Most of your files have comments with some level of detail to make the code understandable at a glance. Some have very detailed comments and explanations throughout the code which is nice to have, but I think not essential for these past files. I've noted which files I would want to see more or expanded comments even if they are brief for now. I think a comment for each select code chunk would be enough. 

For example in this file, I would just be looking for "SQL 1" and "SQL 2" to be replaced with a brief comment of what each of these 2 chunks is doing and/or why there was a need for 2 of them.

File: Code - AdHoc Requests/2020/trello-rafv1.sql [hunk 2]
```diff
@@ -0,0 +1,65 @@
+#Trello Request https://trello.com/c/BlWpOccA/1656-quick-raf-assessment
+
+select action_type, channel, type from public.member_event_log mel where timestamp >= '2019-09-01'
+group by 1,2,3 limit 100
+
+
+_________________
+
+
+select case when (users.source like '%phoenix%' and users.source_detail LIKE '%referrer^_user^_id%' ESCAPE '^') then 'friend-referral'
```
Comment (kkennovin) [inline Code - AdHoc Requests/2020/trello-rafv1.sql:10]: add comments for each select chunk

File: Code - AdHoc Requests/2021/2021-02 unsub_transactional_emails.sql [hunk 3]
```diff
@@ -0,0 +1,47 @@
+-- Ad Hoc Request from Diego
+-- How many users have received transactional emails after unsubscribing in the last 6 months
+-- https://trello.com/c/uVXqTguF/1740-ad-hoc-data-support-unsub-members-receiving-transactional-emails-after-unsub
+
+with
+unsub_members as
+	-- all members who unsubscribed from email or did not optin in the last 6 months
```
Comment (kkennovin) [inline Code - AdHoc Requests/2021/2021-02 unsub_transactional_emails.sql:7]: add comments for each select chunk similar to this one

File: Code - SQL Transforms/Looker/signup_tiers.sql [hunk 4]
```diff
@@ -2,12 +2,12 @@ SELECT
 	count(distinct a.northstar_id),
 	a.total_signups
 FROM
-	(SELECT 
+	(SELECT
```
Comment (kkennovin) [inline Code - SQL Transforms/Looker/signup_tiers.sql:5]: add comments for each select chunk and possibly remove one? there is a "vFINAL" comment in the middle of the 2 chunks but it's unclear if that means one chunk was a draft and is no longer needed.

**Review by kkennovin — COMMENTED** [review]
Thanks for adding the trello links! Otherwise, I have requested brief comments or expanded comments on some files for future clarity.

File: Code - SQL Transforms/Looker/signup_tiers.sql (hunk 4 above)
Comment (ghangas-ds) [reply Code - SQL Transforms/Looker/signup_tiers.sql:5]: I'm unable to make updates to this one. I believe this code was just something I reorganized vs. uploaded new. This code was either Megha's or Eris

**Review by kkennovin — APPROVED** [review]

**Pull request merged**